docs/assets/docs.scss

```diff
@@ -0,0 +1,114 @@
+$primary: #3273dc;
+$primary-darken: #2366d1;
+$light: #f5f5f5;
+$grey: #4a4a4a;
+$white: #fff;
+
+body {
+  font-family: BlinkMacSystemFont, -apple-system, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, Helvetica, Arial, sans-serif;
+  font-weight: lighter;
+  margin: 0;
+  text-rendering: optimizeLegibility;
+}
+
+.container {
+  padding: 1rem 10vw;
+}
+
+.navbar {
+  background-color: $primary;
+
+  &,
+  .navbar__navbar-end {
```

Comment:
A few thoughts on BEM here, that can be applied to all of your BEM:

* Because these are part of the `navbar` block, we don't need to repeat the name in the element, so this can be `navbar__end`.
* Sass `&` can be used in a really nifty way here for working with BEM. Instead of repeating `.navbar`, we can do `&__end` to generate the class `.navbar__end`. When working with BEM, it's OK that that class isn't scoped to the `.navbar` class; the naming convention is what's used to scope.